README.md

```diff
@@ -1,6 +1,6 @@
 # FSH Online
 
-FSH Online is a web application for authoring [FHIR Shorthand (FSH)](https://build.fhir.org/ig/HL7/fhir-shorthand/) and running the [SUSHI](https://github.com/FHIR/sushi) compiler on the authored FSH directly in a web browser. It is available on https://fshschool.org/FSHOnline/.
+FSH Online is a web application for authoring [FHIR Shorthand (FSH)](https://build.fhir.org/ig/HL7/fhir-shorthand/) and running the [SUSHI](https://github.com/FHIR/sushi) compiler on the authored FSH directly in a web browser. It also runs the [GoFSH](https://github.com/FHIR/GoFSH) decompiler to translate FHIR definitions into FSH. It is available on https://fshschool.org/FSHOnline/.
 
 NOTE: This application is an early alpha release and is designed to be used for initial exploration.
```

Comment:
Not saying we need to remove this here, but I am wondering what has to happen for us to no longer consider this "alpha"?

Reply:
Ah that's a good question. I'm not entirely sure either. There's no real versioning so I think this reference is the only place we call it an alpha. I'd feel comfortable taking that line out when we release the `gofsh` branch, but we can also discuss as a team!